package.json

```diff
@@ -24,6 +24,7 @@
     "@fastify/basic-auth": "^5.0.0",
     "fastify": "^4.21.0",
     "fastify-plugin": "^4.5.1",
+    "node": "^20.11.0",
```

Comment:
We'll want to update the `engines` key in the package and `npm uninstall` this `node` from the dependencies.

The engines should look like:
```json
"engines": {
  "node": "^20.x.x"
}
```

Reply:
Hmm, OK. How's this?